CalendarPicker/DaysGridView.js

```diff
@@ -45,7 +45,7 @@ export default class DaysGridView extends Component {
       const firstWeekDay = getISODay(firstDayOfMonth);
 
       // Determine starting index based on first day of week prop.
-      const startIndex = (firstDay > 0) ? (firstWeekDay + Utils.FIRST_DAY_OFFSETS[firstDay]) % 7 : firstWeekDay;
+      const startIndex = (firstDay > 0) ? (firstWeekDay + Utils.FIRST_DAY_OFFSETS[firstDay]) % 7 : firstWeekDay % 7;
```

Comment:
It would be clearer to move the mod outside the ternary since it applies to both cases.
```
const startIndex = ((firstDay > 0) ? (firstWeekDay + Utils.FIRST_DAY_OFFSETS[firstDay]) : firstWeekDay) % 7;
```

Reply:
@peacechen Please review, I have updated code as per suggestion.

Reply:
Thanks @shubhamjadon